packages/commons/src/conditionals/validate.ts

```diff
@@ -59,9 +58,9 @@ export function areConditionsMet(
   conditions: FieldConditional[],
   values: Record<string, unknown>
 ) {
-  return conditions.every((condition) =>
-    isConditionMet(condition.conditional, values)
-  )
+  return conditions.every((condition) => {
+    return ajv.validate(condition.conditional, values)
```

Comment:
This is modifying existing functionality that conditionals in the forms use. Are you sure you're not breaking it?